server/src/main/java/org/apache/kafka/server/config/ShareGroupConfig.java

```diff
@@ -63,14 +63,19 @@ public class ShareGroupConfig {
     public static final int SHARE_GROUP_MAX_RECORD_LOCK_DURATION_MS_DEFAULT = 60000;
     public static final String SHARE_GROUP_MAX_RECORD_LOCK_DURATION_MS_DOC = "The record acquisition lock maximum duration in milliseconds for share groups.";
 
+    public static final String SHARE_FETCH_PURGATORY_PURGE_INTERVAL_REQUESTS_CONFIG = "share.fetch.purgatory.purge.interval.requests";
```

Comment:
This config is not in the KIP. So we need to update the KIP.

Reply:
I have notified @AndrewJSchofield about this new config. He will add it in the KIP.

Reply:
Config added to the KIP.